source/Calamari.Tests/Fixtures/Integration/Proxies/ProxyEnvironmentVariablesGeneratorFixture.cs

```diff
@@ -232,7 +232,7 @@ void AssertProxyBypassed(IEnumerable<EnvironmentVariable> result)
         
         void AssertUnauthenticatedSystemProxyUsed(IEnumerable<EnvironmentVariable> output)
         {
-#if !NETCOREAPP3_1
+#if !NETCOREAPP
```

Comment:
These constants are usually defined in the relevant `csproj`. It looks like at some stage, things got out of sync. 

The [constant I can see](https://github.com/OctopusDeploy/Calamari/pull/933/files#diff-68d19028bf7ced97e653357fa76afc2adf2ac2be0bd449d507a8d20e639c3375R21) which would be defined for `net6.0` is `NETCORE` rather than `NETCOREAPP`. We probably need to use that here for it to work as you expect.